legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -43,6 +47,8 @@
                     <span> - </span>
                     {% if business.legalType in ['GP', 'SP'] and business.state == 'HISTORICAL' %}
                         <span>Dissolved</span>
+                    {% elif business.legalType in ['C', 'CCC', 'CUL', 'CBEN'] and business.state == 'HISTORICAL' %}
+                        <span>Historical - Dissolved for Failure to File on {{stateFilings[0].effectiveDateTime}}</span>
```

Comment:
`Historical - ` not required. line no 45-47 will include this, and i am not sure about this statement. A business can be dissolved due to multiple reason (may not only because of failure to file). Please verify this with Naveen

Reply:
Did you verify if this statement is correct?

Reply:
Waiting for Naveen's response

Reply:
You can merge once its confirmed